### PR TITLE
Add Expand Capability to pfNotificationDrawer

### PR DESCRIPTION
**Pull request by jeff-phillips-18**

### CODE REVIEW

**Comment by jeff-phillips-18** [comment]
@dtaylor113 @dlabrecq @dgutride Please review

File: src/notification/notification-drawer.directive.js [hunk 1]
```diff
@@ -14,7 +14,9 @@
  *   <br><br>
  *
  * @param {boolean} drawerHidden Flag if the drawer is currently hidden
- * @param {string}  drawerTitle  Title to display for the drawer
+ * @param {boolean} allowexpand Flag if the drawer can be expanded. Optional, default: false
```
Comment (dgutride) [inline src/notification/notification-drawer.directive.js]: Should this be allowExpand for consistency?

**Comment by dgutride** [comment]
Looks good (other than the one param documentation note above)

**Comment by jeff-phillips-18** [comment]
Fixed typo in documentation ( @dgutride issue )

**Comment by jeff-phillips-18** [comment]
Added a bit more demo capabilities (mark read on click, implement Mark All Read, Clear All)

**Comment by priley86** [comment]
Looks great @jeff-phillips-18 ! LGTM 👍

**Comment by jeff-phillips-18** [comment]
Added a touch of css cleanup to prepare classes to be in base patternfly.

**Comment by dtaylor113** [comment]
LGTM.  I'd like to suggest an MiQ PR practice of including a screenshot of the area of the GUI/widget which is being updated.

**Comment by dgutride** [comment]
Looks good to me.

**Pull request merged**